configs/shipitworker

```diff
@@ -0,0 +1,24 @@
+{
+    "us-east-1": {
+        "type": "shipitworker",
+        "domain": "srv.releng.use1.mozilla.com",
+        "ami": "ami-4dc07a26",
+        "subnet_ids": ["subnet-33a98358"], // TODO
+        "security_group_ids": ["sg-e3061897"],
```

Comment:
TODO should be on the security group rather than subnet, I think.

Reply:
That's correct. I wasn't sure of the subnet, but NetOps was okay with it in [bug 1432838](https://bugzilla.mozilla.org/show_bug.cgi?id=1432838). I also confirm this security group is a new one, dedicated to shipit.